lib/measures/add_ems_emissions_reporting/measure.rb

```diff
@@ -335,16 +361,48 @@ def run(model, runner, usr_args)
     ems_prgm.addLine('SET his_yr = (his_yr_val * elec / conv) * conv_kg_mt')
 
+    ##### Emisison factors for natural gas, propane, and fuel oil based on EPA eGRID data and calculated using 20-year GWP horizon based on ASHRAE 189.1
+    ## natural gas :  277.358126 KG/MWH
+    ## propane : 323.896704 KG/MWH
+    ## Fuel oil : 294.962046 KG/MWH
+    # nat gas
+    ems_prgm.addLine('SET nat_gas_val = 277.358126') 
+    ems_prgm.addLine('SET nat_gas = Gas_Sen')
+    ems_prgm.addLine('SET conv = 1000000 * 60 * 60') # J to MWh (1000000J/MJ * 60hr/min * 60 min/sec)
+    ems_prgm.addLine('SET conv_kg_mt = 0.001') # kg to metric ton
+    ems_prgm.addLine('SET nat_gas_em = (nat_gas_val * nat_gas / conv) * conv_kg_mt')
+    # lpg
+    ems_prgm.addLine('SET lpg_val = 323.896704')
+    ems_prgm.addLine('SET lpg = Lpg_Sen')
+    ems_prgm.addLine('SET conv = 1000000 * 60 * 60') # J to MWh (1000000J/MJ * 60hr/min * 60 min/sec)
+    ems_prgm.addLine('SET conv_kg_mt = 0.001') # kg to metric ton
+    ems_prgm.addLine('SET lpg_em = (lpg_val * lpg / conv) * conv_kg_mt')
+    # fo1
+    ems_prgm.addLine('SET fo1_val = 294.962046')
+    ems_prgm.addLine('SET fo1 = Fo1_Sen')
+    ems_prgm.addLine('SET conv = 1000000 * 60 * 60') # J to MWh (1000000J/MJ * 60hr/min * 60 min/sec)
+    ems_prgm.addLine('SET conv_kg_mt = 0.001') # kg to metric ton
+    ems_prgm.addLine('SET fo1_em = (fo1_val * fo1 / conv) * conv_kg_mt')
+    # fo2
+    ems_prgm.addLine('SET fo2_val = 294.962046')
+    ems_prgm.addLine('SET fo2 = Fo2_Sen')
+    ems_prgm.addLine('SET conv = 1000000 * 60 * 60') # J to MWh (1000000J/MJ * 60hr/min * 60 min/sec)
+    ems_prgm.addLine('SET conv_kg_mt = 0.001') # kg to metric ton
+    ems_prgm.addLine('SET fo2_em = (fo2_val * fo2 / conv) * conv_kg_mt')
 
     #### add emissions intensity metric
     # get building from model 
     building = model.getBuilding
     floor_area = building.floorArea * 10.764 #change from m2 to ft2
     #add metric
-    ems_prgm.addLine("SET fut_hr_intensity = fut_hr * 1000 / #{floor_area}") # unit: kg/ft2 - changed mt to kg
-    ems_prgm.addLine("SET his_hr_intensity = his_hr * 1000 / #{floor_area}") # unit: kg/ft2 - changed mt to kg
-    ems_prgm.addLine("SET fut_yr_intensity = fut_yr * 1000 / #{floor_area}") # unit: kg/ft2 - changed mt to kg
-    ems_prgm.addLine("SET his_yr_intensity = his_yr * 1000 / #{floor_area}") # unit: kg/ft2 - changed mt to kg
+    ems_prgm.addLine("SET fut_hr_intensity = fut_hr / #{floor_area}")
+    ems_prgm.addLine("SET his_hr_intensity = his_hr / #{floor_area}")
+    ems_prgm.addLine("SET fut_yr_intensity = fut_yr / #{floor_area}")
+    ems_prgm.addLine("SET his_yr_intensity = his_yr / #{floor_area}")
+    ems_prgm.addLine("SET nat_gas_em_intensity = nat_gas_em / #{floor_area}")
+    ems_prgm.addLine("SET lpg_em_intensity = lpg_em / #{floor_area}")
+    ems_prgm.addLine("SET lpg_em_intensity = fo1_em / #{floor_area}")
+    ems_prgm.addLine("SET lpg_em_intensity = fo2_em / #{floor_area}")
```

Comment:
can we add unit comments back in? I think you said mt/ft2

Reply:
yeah add units, but intensities are reported in kgCO2/ft2/year.